### PR TITLE
HEEDLS-330 Post learning assessment fullscreen

### PR DESCRIPTION
**Pull request by kwunkiuma**
Add fullscreen functionality to post learning content page.

The add responsive resizing; maintaining a 4:3 ratio.

Modify typescript to work after entering fullscreen.

Add tests for close page function after entering fullscreen.

Issue with padding on sides is also fixed.

Ran and passed all unit tests, screenshots taken in Google Chrome.

There is an issue on Firefox where the height of content inside the iframe is too big.

Browser
image

Browser fullscreen
image

Mobile
image

Mobile fullscreen
image

### CODE REVIEW

File: DigitalLearningSolutions.Web/Views/LearningMenu/PostLearning/PostLearningContent.cshtml [hunk 1]
```diff
@@ -39,11 +40,25 @@
...
 
-    @{ var contentType = "assessment"; }
-    <partial name="Shared/_JavaScriptDisabledError" model="contentType" />
+@{ var contentType = "assessment"; }
+<partial name="Shared/_JavaScriptDisabledError" model="contentType" />
 
-    <iframe src="@Model.ContentSource" class="nhsuk-grid-column-full nhsuk-u-margin-bottom-6 js-only-block" height="800">
-    </iframe>
-  </div>
+<div class="nhsuk-u-margin-bottom-6 content-viewer_iframe-wrapper js-only-block" id="content-viewer_iframe-wrapper">
```
Comment (FridaTveit) [inline DigitalLearningSolutions.Web/Views/LearningMenu/PostLearning/PostLearningContent.cshtml:49]: If this is exactly the same for all three content viewers, might be worth pulling out to a partial? 🙂

**Review by FridaTveit — COMMENTED** [review]
One small suggestion, otherwise looks good 🙂

File: DigitalLearningSolutions.Web/Views/LearningMenu/PostLearning/PostLearningContent.cshtml (hunk 1 above)
Comment (kwunkiuma) [reply DigitalLearningSolutions.Web/Views/LearningMenu/PostLearning/PostLearningContent.cshtml:49]: I assume this should contain both the iframes and buttons. What would be a good name for this partial?

Comment (FridaTveit) [reply DigitalLearningSolutions.Web/Views/LearningMenu/PostLearning/PostLearningContent.cshtml:49]: Good question. Maybe something like `_IFrameWithFullscreen`? 🙂

**Review by FridaTveit — APPROVED** [review]
Looks good 🙂

**Pull request merged**